i18n: Add translation (#27)
KevinMX/support-matrix: Duo_S/BuildRoot/README.md

### Hardware Information

- Milk-V Duo S (512M, SG2000)
- A USB Power Adapter
- A USB-A to C or USB C to C Cable for powering the development board
- A microSD Card
- A USB Card Reader

> - A USB to UART Debugger (e.g., CP2102, FT2232, etc., take note not to use CH340/341 series as they may cause garbled text)

Reviewer: ```suggestion
- A USB to UART Debugger (e.g., CP2102, FT2232, etc. Be aware that WCH CH340/341 series will cause garbled text output, DO NOT USE)
```
